client/src/pages/SignUp/SignUp.tsx

```diff
@@ -32,6 +33,7 @@ export default function Register(): JSX.Element {
         updateSnackBarMessage(data.error.message);
       } else if (data.success) {
         updateLoginContext(data.success);
+        newCust();
```

Comment:
Couldn't we technically just do this logic on the back-end when they sign up?

Reply:
Latest version now has `auth/register` controller call stripe's `createCustomer` controller. Removed this logic from the front-end altogether.